Formula/i/include-what-you-use.rb

```diff
@@ -26,7 +25,7 @@ class IncludeWhatYouUse < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "llvm@18"
+  depends_on "llvm"
```

Comment:
I see `llvm` is currently at 19.1.3. Is it possible to pin to `llvm@19` to avoid the snapshot (or whatever the term is in homebrew) version to jump to @20? IWYU is not guaranteed to work well across LLVM versions.

Reply:
understandable, we will switch to use llvm@19 when llvm 20 got released.